guides/v2.1/install-gde/prereq/mysql.md

```diff
@@ -30,6 +30,11 @@ Magento _strongly_ recommends you observe the following standard when you set up
 *	Familiarize yourself with [these potential MySQL trigger limitations](http://dev.mysql.com/doc/mysql-reslimits-excerpt/5.1/en/stored-program-restrictions.html){:target="_blank"} before you continue.
 *	If you use MySQL database replication, be aware that Magento does _not_ support MySQL statement-based replication. Make sure you use _only_ [row-based replication](http://dev.mysql.com/doc/refman/5.1/en/replication-formats.html){:target="_blank"}.
 
+{:.bs-callout .bs-callout-warning} Magento 2 currently utilizes `CREATE TEMPORARY TABLE` statements inside transactions,
+which are [incompatible](https://dev.mysql.com/doc/refman/5.7/en/replication-gtids-restrictions.html) with database
+implementations utilizing GTID-based replication.
+(E.g., [Google Cloud SQL second-generation instances](https://cloud.google.com/sql/docs/features#differences).)
```

Comment:
Global for all versions:

Change "replication. (E.g.," to "replication, such as "

Reply:
Done.